tests/data/validation/enum/enum.go

```diff
@@ -296,6 +366,16 @@ func (j *EnumMyNumberTypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyNumberTypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyNumberTypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?